ipython/Enwiki Revert Classifier.ipynb

```diff
@@ -0,0 +1,301 @@
+      "          ores.models.enwiki.features > \\\n",
+      "      datasets/enwiki.features_reverted.5k.tsv\n",
+      "\n",
+      "A few of the revisions won't be found -- which is fine.  We'll just have to work with the remaining data.  So, let's load it in!"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "feature_scores = []\n",
+      "with open(\"../datasets/enwiki.features_reverted.5k.tsv\") as f:\n",
+      "    for line in f:\n",
+      "        parts = line.strip().split(\"\\t\")\n",
+      "        values = parts[:-1]\n",
+      "        reverted = parts[-1] == \"True\"\n",
+      "        \n",
+      "        feature_values = []\n",
+      "        for feature, value in zip(features, values):\n",
+      "            \n",
+      "            if feature.returns == bool:\n",
+      "                feature_values.append(value == \"True\")\n",
+      "            else:\n",
+      "                feature_values.append(feature.returns(value))\n",
+      "    \n",
+      "        feature_scores.append((feature_values, reverted))\n",
+      "\n",
+      "len(feature_scores)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": [
+      {
+       "metadata": {},
+       "output_type": "pyout",
+       "prompt_number": 2,
+       "text": [
+        "4982"
+       ]
+      }
+     ],
+     "prompt_number": 2
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "## Training the models\n",
+      "\n",
+      "Now we'll use the extracted features to train some models.  For this section, I'll use an RBF and Linear kernel Support Vector Classifiers."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "from ores.models.enwiki import linear_svc, rbf_svc\n",
+      "from random import shuffle\n",
+      "\n",
+      "shuffle(feature_scores) # Randomize the set\n",
+      "\n",
+      "train_set = feature_scores[1000:] # Nearly 4000 training observations\n",
+      "test_set = feature_scores[:1000]  # 1000 test observations\n",
+      "\n",
+      "print(linear_svc.train(train_set))\n",
+      "print(rbf_svc.train(train_set))"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": [
+      {
+       "output_type": "stream",
+       "stream": "stdout",
+       "text": [
+        "{'seconds_elapsed': 13.189335584640503}\n",
+        "{'seconds_elapsed': 4.040912389755249}"
+       ]
+      },
+      {
+       "output_type": "stream",
+       "stream": "stdout",
+       "text": [
+        "\n"
+       ]
+      }
+     ],
+     "prompt_number": 3
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "## Testing the models\n",
+      "\n",
+      "First, the linear kernel."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "import matplotlib.pyplot as plt\n",
```

Comment:
At first, I got `ImportError: No module named 'matplotlib'`. Then I executed `pip install matplotlib` in the terminal but an error occurred: http://dpaste.com/35AM0JK.